compiler/cpp/test/util/FileReader.cpp

```diff
@@ -0,0 +1,16 @@
+//
+// Created by Marc on 30.05.2021.
+//
+
+#include "FileReader.h"
+
+std::string FileReader::fileToString(const std::string& subDir, const std::string& fileName) {
+    std::ifstream file(dirPath + "/" + subDir + "/" + fileName);
+    std::string content;
+    if (file) {
+        std::ostringstream stringStream;
+        stringStream << file.rdbuf();
+        content = stringStream.str();
+    }
+    return content;
```

Comment:
Is this compliant to the rest? That if there is not a file, you return `null`.

Reply:
You're right. Fixed it by throwing an error when the file could not be read.